hexa/pipelines/migrations/0043_generate_webhook_token.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 5.0.3 on 2024-06-04 20:59
+import base64
+
+from django.core.signing import TimestampSigner
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    def create_webhook_token(apps, schema_editor):
```

Comment:
Some people are using the webhooks already. How will you handle a smooth transition for them? I think in that case we have to keep the url as it is and warn them (export a list of the editor/admin members of the pipelines' workspaces)

Reply:
So return the keep old URL(/pipelines/uuid/run) and be able to use it alongside the new system and deprecated later?

Reply:
Yes or leave the url as it is and set the uuid as the webhook_token